specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -6817,6 +6817,10 @@
           "$ref": "#/definitions/VirtualMachineScaleSetIpTag"
           },
         "description": "The list of IP tags associated with the public IP address."
+        },
+       "publicIPPrefix": {
```

Comment:
This API of Compute has already been released. Are you saying that this field is already available, or you will add it later at the same time Network 2018-07-01 goes public?
Please tag someone from VMSS team to confirm this.